### PR TITLE
Morgue and Crematorium Directional Sprites + Number of related fixes (Retrying #35068)

### PR DESCRIPTION
**Pull request by Chuckleberry-Finn**
Should be identical to: #35068 - but actually merge-able.

[Changelogs]:
:cl: Jittai
imageadd: Crematoriums and Morgue Trays now have directional states.
fix: Morgue Trays now checks for cloneable people like the dna-scanner does. (ghosts can roam now)
fix: Cremated mice no longer leave behind snackable bodies.
fix: Ash does not get instantly deleted in the crematorium upon creation. Ash also piles up. (aesthetically only)
tweak: Morgue/Crematorium Trays make a slight rolling sound.
/:cl:

m
mmm

[why]: Trays can now be back to back saving space, also not always facing East/South.

### CODE REVIEW

**Comment by ChuckTheSheep** [comment]
Slightly tweaked the lights down (as ShizCalev pointed out).

**Comment by ExcessiveUseOfCobblestone** [comment]
really nice

**Comment by StreakyHaddock** [comment]
oh god yes

**Comment by PKPenguin321** [comment]
We've needed directional morgue trays for a while, I'm sure there's an issue report regarding it somewhere. Really nice

**Comment by vuonojenmustaturska** [comment]
image


Considering this and that this PR is a re-do of #35068 which had dmi conflict issues, i'll just press a button

**Comment by Bawhoppen** [comment]
An honest to goodness PR like this is rare nowadays

File: code/game/objects/effects/decals/cleanable.dm [hunk 1]
```diff
@@ -21,7 +21,8 @@
 				diseases_to_add += D
 		if(LAZYLEN(diseases_to_add))
 			AddComponent(/datum/component/infective, diseases_to_add)
-	. = ..()
+
+	return//. = ..()//cleanable stuff sometimes needs to be in objs
```
Comment (Cyberboss) [inline code/game/objects/effects/decals/cleanable.dm]: y u do this >:[

**Pull request merged**